Creature.playground/Contents.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+//Creation of Creature Structure
+
+struct Creature{
+    var name:String
```

Comment:
The Swift community is very picky about style. Most organizations I have worked with follow a Swift Style Guide based Google, AirBNB, or Realm. Realm is the basis for the very popular Swift linter called SwiftLint.

Variables and let constants should have a colon (:) immediately following the name and a space between the colon and the type.

var description:String 

var description: String

This style should also be used in your parameters.